README.md

```diff
@@ -339,27 +339,17 @@ db_path = omero2pandas.upload_table("/path/to/my_data.csv", "Name for table",
 # Returns the path to the created tiledb file
 ```
 
-Similar to regular table uploads, the input can be a dataframe in memory or a 
-csv file on disk. The input will be copied into a new TileDB database and 
-registered to OMERO in-place.
+This will convert the table into a TileDB database which will be written to `local_path`, then attempt to 
+register this to OMERO in-place. For this to work the local_path needs to be visible on the server machine 
+as well (e.g. a network drive).
```

Comment:
```suggestion
This will convert the table into a TileDB file written to `local_path`, then attempt to 
register this table to OMERO Plus in-place. For this to work the `local_path` needs to be readable via the server machine 
as well (e.g. a network drive).
```